_benchmark/user-guide/understanding-results.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: Understanding benchmark results
+nav_order: 22
+parent: User guide
+---
+
+
+At the end of each test run, OpenSearch Benchmark creates a summary of test results metrics like service time, throughput, latency, and more. These metrics provide insights into how the workload selected performed on a benchmarked OpenSearch cluster.
+
+The following guide gives information about how to understand the results of the summary report.
+
+## OpenSearch Benchmark runs
+
+OpenSearch Benchmark runs a series of nightly tests targeting the OpenSearch development cluster. The results of these test runs can be found on https://opensearch.org/benchmarks. The results show the metrics that span different test runs and target both recent and future versions of OpenSearch.
```

Comment:
Can we define 'runs' here if that is the terminology that we are using for the nightly test? OpenSearch Benchmark runs are the nightly tests that target the OS dev cluster..."